--- emf-2.6.0/org.eclipse.emf.sdk-feature/build.properties	2010-06-01 12:00:18.000000000 -0400
+++ emf-2.6.0/org.eclipse.emf.sdk-feature/build.properties.adddocs	2010-08-03 16:30:45.000000000 -0400
@@ -7,3 +7,4 @@
                license.html
 root=rootfiles
 generate.feature@org.eclipse.emf.source=org.eclipse.emf.sdk, exclude@org.eclipse.emf.doc, exclude@org.eclipse.emf.example.installer, exclude@org.eclipse.emf.mapping.ecore, exclude@org.eclipse.emf.mapping.ecore.editor
+generate.feature@org.eclipse.emf.doc.source=org.eclipse.emf.doc
--- emf-2.6.0/org.eclipse.xsd.sdk-feature/build.properties	2010-06-01 12:40:35.000000000 -0400
+++ emf-2.6.0/org.eclipse.xsd.sdk-feature/build.properties.build-converter	2010-08-03 17:05:57.845492781 -0400
@@ -7,3 +7,10 @@
                license.html
 root=rootfiles
 generate.feature@org.eclipse.xsd.source=org.eclipse.xsd.sdk, exclude@org.eclipse.xsd.cheatsheets, exclude@org.eclipse.xsd.doc, exclude@org.eclipse.xsd.ecore.converter, exclude@org.eclipse.xsd.example.installer, exclude@org.eclipse.xsd.mapping, exclude@org.eclipse.xsd.mapping.editor
+generate.feature@org.eclipse.xsd.ecore.converter.source=org.eclipse.xsd.ecore.converter
+generate.feature@org.eclipse.xsd.edit.source=org.eclipse.xsd.edit
+generate.feature@org.eclipse.xsd.editor.source=org.eclipse.xsd.editor
+generate.feature@org.eclipse.xsd.mapping.source=org.eclipse.xsd.mapping
+generate.feature@org.eclipse.xsd.mapping.editor.source=org.eclipse.xsd.mapping.editor
+generate.feature@org.eclipse.xsd.doc.source=org.eclipse.xsd.doc
+individualSourceBundles=true
