--- emf-2.6.0/org.eclipse.xsd.sdk-feature/feature.xml	2010-05-27 14:12:20.000000000 -0400
+++ emf-2.6.0/org.eclipse.xsd.sdk-feature/feature.xml.noxsd2ecore	2010-08-04 15:06:18.479696787 -0400
@@ -30,8 +30,6 @@
 
    <includes id="org.eclipse.xsd" version="0.0.0"/>
    <includes id="org.eclipse.xsd.source" version="0.0.0"/>
-   <includes id="org.eclipse.xsd.ecore.converter" version="0.0.0"/>
-   <includes id="org.eclipse.xsd.ecore.converter.source" version="0.0.0"/>
    <includes id="org.eclipse.xsd.edit" version="0.0.0"/>
    <includes id="org.eclipse.xsd.edit.source" version="0.0.0"/>
    <includes id="org.eclipse.xsd.editor" version="0.0.0"/>
